metacat-main/src/main/java/com/netflix/metacat/main/api/v1/MetacatController.java

```diff
@@ -578,7 +580,10 @@ public DatabaseDto getDatabase(
         return this.requestWrapper.processRequest(
             name,
             "getDatabase",
-            () -> databaseService.get(name, includeUserMetadata, includeTableNames)
+            () -> databaseService.get(name,
+                GetDatabaseParameters.builder()
```

Comment:
Isn't GetDatabaseRequest more appropriate?

Reply:
This is the internal parameter. We have the getTableRequest xx like class for the calls from outside.

Reply:
GetDatabaseServiceRequest or GetDatabaseServiceParameters?

Reply:
Sure, that's reasonable. I will rename to GetDatabaseServiceParameters